documentation/STRUCTURE.md

```diff
@@ -0,0 +1,161 @@
+- [Mitigating Disparities in Ranking from Binary Data](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Mitigating%20Disparities%20in%20Ranking%20from%20Binary%20Data.ipynb)
+
+### Community
+
+This page should provide basic information about how to reach out in order to ask questions, provide feedback, report bugs or request features. Similarly, it should be clear how people can reach out if they want to contribute to Fairlearn, and where they can find small items to get started. [This](https://scikit-learn.org/dev/developers/contributing.html) is an example of how scikit-learn handles it through a contribution guide that is somewhat similar to ours that we currently have in the repo.
+
+We could also write up the history of the project, its roadmap, and how we envision governance to work. The code of conduct should certainly be mentioned as well.
+
+Some projects have a page showing the maintainers as well:
+
+- [scikit-learn](https://scikit-learn.org/stable/about.html#people)
+- [pandas](https://pandas.pydata.org/about/team.html)
+
+## Required steps
+
+To properly format math, text, and links across the pages we need to use ReST. Some of our existing documentation may have to be converted from Markdown to ReST, e.g. the terminology page.
+
+We need to find a way to present the dashboard in a website where it can't be interactive. Perhaps with screenshots for the user guides, but the case studies are downloadable as Jupyter notebooks. [Could we perhaps pre-calculate all metrics and show the interactive dashboard in the case studies?]
+
+We already have the examples gallery thanks to Adrin's work on sphinx-gallery: https://fairlearn.readthedocs.io/en/latest/index.html
+There will be plenty of work to
+
+- convert existing notebooks to case studies (or user guides)
+- write all user guides
+- write the installation guide
+
+The repository structure should look similar to [what scikit-learn has](https://github.com/scikit-learn/scikit-learn/tree/master/doc):
+
+- top-level doc directory has all the ReST files for the webpage except for API documentation and case studies
+- API documentation comes directly from the code documentation
+- case studies live in a separate top-level directory (scikit-learn calls it `examples`) as python files
+
+Putting it all together:
+
+- deploy through fairlearn.org
+  - landing page will be provided by a designer, everything else from the Fairlearn project repositories
+  - CI should generate documentation and make it available at PR-time for reviewers to check
+  - all pages are reachable through fairlearn.org
+  - https for fairlearn.org (currently only http works)
+  - remove API doc from readthedocs
+- ensure the navigation from homepage to the other sections works
+  - manual testing?
+  - automated testing? (also check for broken links)
```

Comment:
we're not too picky about this. We check for sphinx warnings and errors which also catch the missing links sometimes, but a big part of our external contributions are on the documentation. Documentation is always a good place for externals to start contributing, so they definitely don't have to be perfect ;)

Reply:
Do you have any spiders to find broken links? When I was first working on the documentation, I was sometimes very frustrated by links not working for reasons which were often obscure.

Reply:
If we use the rst format and link to different places using its syntax, sphinx will automatically warn if those references are broken.